docs/publish/publish.md

```diff
@@ -45,6 +45,12 @@ The deployment options that are available depend on the Office application that
 
 The following sections provide additional information about the deployment methods that are most commonly used to distribute production Office Add-ins to users within an organization.
 
+> [!IMPORTANT]
```

Comment:
I'd be tempted to not put this in a note block so you could make the bulleted list an include file.

Reply:
fixed.